tgstation.dme

```diff
@@ -2897,6 +2897,7 @@
 #include "modular_citadel\code\modules\events\blob.dm"
 #include "modular_citadel\code\modules\events\wizard\magicarp.dm"
 #include "modular_citadel\code\modules\food_and_drinks\snacks\meat.dm"
+#include "modular_citadel\code\modules\integrated_electronics\subtypes\input.dm"

```

Comment:
modularity is no longer necessary since we no longer use a mirror bot

Reply:
So should I move it the main input file?